layers/+music/tidalcycles/packages.el

```diff
@@ -0,0 +1,48 @@
+;;; packages.el --- TidalCycles Layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Riccardo Binetti <rbino@gmx.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst tidalcycles-packages
+  '(tidal))
+
+(defun tidalcycles/init-tidal ()
+  (use-package tidal
```

Comment:
Please try to defer the loading of the package. If possible all packages within spacemacs should be lazy loaded to avoid long start-up times. If the package does not have proper autoloads you can try to add these via use-package.

If it is not working we can keep it as it is, but we should at least try.